src/setup/mouse.c

```diff
@@ -102,8 +104,10 @@ static void ConfigExtraButtons(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
     AddMouseControl(buttons_table, "Move backward", &mousebbackward);
     AddMouseControl(buttons_table, "Strafe right", &mousebstraferight);
     AddMouseControl(buttons_table, "Previous weapon", &mousebprevweapon);
-    AddMouseControl(buttons_table, "Strafe on", &mousebstrafe);
+    AddMouseControl(buttons_table, "Turn left", &mousebturnright);
     AddMouseControl(buttons_table, "Next weapon", &mousebnextweapon);
+    AddMouseControl(buttons_table, "Turn right", &mousebturnright);
+    AddMouseControl(buttons_table, "Strafe on", &mousebstrafe);
```

Comment:
Why did you move "Strafe On"? Why aren't the two new ones next to each other?

Reply:
So they align up more nicely in the Mouse menu in chocolate-doom-setup.

Reply:
You will note that the other menu items are also aligned like this.

Reply:
I looked at the setup menu. Now "Strafe On" is on the left side, separate from "Strafe Left/Right" which are on the right side. This looks weird. Can you fix it so that they're all on the right? This requires that the right column has more items than left. I'm not sure the text screen code supports that.

Reply:
I don't think that's possible. But we could probably have foreward+backward+strafeleft+straferight+strafeon on the left and turnleft+turnright+nextweap+prevweap on the right side.